### ISSUE TITLE
Carousel: Card doesn't truncate metadata1 correctly when rendered in a carousel

### ISSUE THREAD

**Issue by nttibbetts** [pluralsight/design-system#594]
### Expected behavior
When the items to rendered in the metadata1 is wider than the card, they should get truncated with ellipses 

### Actual behavior
The metadata1 items are allowed to render fully stretching card

![Screen Shot 2019-07-26 at 10 17 20 AM](https://user-images.githubusercontent.com/1699510/61969959-befb8800-afa9-11e9-96a8-3158e08199e4.png)

Here's a comparison for the same card:

 Inside Design System Carousel | Outside Design System Carousel
--- | ---
![in-carousel](https://user-images.githubusercontent.com/1699510/62302595-a1ff0300-b448-11e9-9e51-6889c13b1b01.png) | ![outside-carousel](https://user-images.githubusercontent.com/1699510/62302609-a7f4e400-b448-11e9-887d-b2bdab78c6b0.png)

We were just experimenting with replacing our carousel with the design system carousel but didn't commit the changes. If necessary we could commit this to a branch, but it'd likely be easier to mock it up as it's all design system components.

### Steps to repro

1. Add a card to a carousel and give it 3 or more metadata1 items

### Related packages

- carousel@1.1.1
- card@9.0.7

### Environment

- OS - OSX 10.14.5
- Browser version - Chrome v75

**Comment by jaketrent** [comment]
Triage: Don't quite understand as filed.  Does this issue essentially mean "The cards should all be equal width?"  I'm not seeing metadata2 problems in the screenshot.  I am seeing non-equal card widths.  If we're re-interpreting this correctly, we'll address this.

**Comment by nttibbetts** [comment]
I'll setup a case and get a screenshot tomorrow to shown the intended (and currently supported) behavior of the Card component in the situation I'm describing when it's not being used in the Carousel component and hopefully that'll better describe the problem.

**Comment by nttibbetts** [comment]
Apologies for the confusion, while getting screenshots for the comparison I realized it wasn't the metadata2 prop, but the metadata1 prop, not sure if there's any significant difference in the behavior of those two props. I've updated the issue to correct this.

I've also included screenshots of the same usage of Card inside of the Carousel and outside the Carousel to better highlight the change in behavior. Hopefully that makes the issue a bit clearer.
